package.json

```diff
@@ -48,5 +50,8 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+    "engines": {
+    "node": ">=16.14"
```

Comment:
should we use LTS? Node version 18?

Reply:
will look at this, if it needs doing, next PR